deploy/CVModelSQLScript.sql

```diff
@@ -0,0 +1,5 @@
+CREATE EXTENSION postgis;
+
+CREATE SCHEMA aioutputmodelschema CREATE TABLE cvmodel (id VARCHAR(40), probability NUMERIC, tagid text, tagname text, tagtype text, tile text);
+
+ALTER TABLE aioutputmodelschema.cvmodel ADD COLUMN location geometry(Polygon, 4326);
```

Comment:
Trivial, optional but may be good to just put this in CREATE TABLE script itself instead of separate ALTER script